docs/conf.py

```diff
@@ -29,10 +29,10 @@
 extensions = [
     "sphinx.ext.autodoc",
     "sphinx.ext.intersphinx",
+    "sphinx.ext.extlinks",
```

Comment:
```suggestion
    "sphinx.ext.extlinks",
    "sphinx.ext.intersphinx",
```
Alphabetical sorting.